module3/homework/advanced_calculator/advancedCalculator.cpp

```diff
@@ -0,0 +1,164 @@
+#include <algorithm>
+#include <cmath>
+#include <functional>
+#include <map>
+#include <string>
+
+#include "advancedCalculator.hpp"
+
+const std::string operations = "+-*/%$^!";
+const std::string digits = "1234567890.";
```

Comment:
Good idea to make it const 👍  I would consider moving them to the header.

Reply:
I thought that putting variables declarations/definitions in .hpp files wasn't good idea. Correct me if I'm wrong 🙂

Reply:
I used to put them in the header,because it make them easier to access(in case you would need to modify them),but on the second thought it puts them in every file using that header(so better have them locally). Thanks for question.